schemas/launch_v1.json

```diff
@@ -0,0 +1,121 @@
+  "type": "object",
+  "properties": {
+    "account_service_log_out_url": {
+      "$ref": "common/definitions.json#/account_service_logout_url"
+    },
+    "account_service_url": {
+      "$ref": "common/definitions.json#/account_service_url"
+    },
+    "case_id": {
+      "$ref": "common/definitions.json#/case_id"
+    },
+    "case_ref": {
+      "$ref": "common/definitions.json#/case_ref"
+    },
+    "case_type": {
+      "$ref": "common/definitions.json#/case_type"
+    },
+    "channel": {
+      "$ref": "common/definitions.json#/channel"
+    },
+    "collection_exercise_sid": {
+      "$ref": "common/definitions.json#/collection_exercise_sid"
+    },
+    "display_address": {
+      "$ref": "common/definitions.json#/display_address"
+    },
+    "employment_date": {
+      "$ref": "common/definitions.json#/employment_date"
+    },
+    "eq_id": {
+      "$ref": "common/definitions.json#/form_type",
+      "description": "The EQ questionnaire ID."
+    },
+    "exp": {
+      "description": "JWT Issued At claim, see https://tools.ietf.org/html/rfc7519#section-4.1",
+      "type": "integer"
+    },
+    "form_type": {
+      "$ref": "common/definitions.json#/form_type"
+    },
+    "iat": {
+      "description": "JWT Expiration Time claim, see https://tools.ietf.org/html/rfc7519#section-4.1",
+      "type": "integer"
+    },
+    "jti": {
+      "$ref": "common/definitions.json#/jti"
+    },
+    "language_code": {
+      "$ref": "common/definitions.json#/launch_language_code"
+    },
+    "period_id": {
+      "$ref": "common/definitions.json#/period_id"
+    },
+    "period_str": {
+      "$ref": "common/definitions.json#/period_str"
+    },
+    "ref_p_end_date": {
+      "$ref": "common/definitions.json#/ref_p_end_date"
+    },
+    "ref_p_start_date": {
+      "$ref": "common/definitions.json#/ref_p_start_date"
+    },
+    "region_code": {
+      "$ref": "common/definitions.json#/region_code"
+    },
+    "response_expires_at": {
+      "$ref": "common/definitions.json#/response_expires_at"
+    },
+    "response_id": {
+      "$ref": "common/definitions.json#/response_id"
+    },
+    "ru_name": {
+      "$ref": "common/definitions.json#/ru_name"
+    },
+    "ru_ref": {
+      "$ref": "common/definitions.json#/ru_ref"
+    },
+    "schema_name": {
+      "$ref": "common/definitions.json#/schema_name"
+    },
+    "survey": {
+      "$ref": "common/definitions.json#/non_empty_string",
+      "description": "[DEPRECATED] The survey being launched."
+    },
+    "trad_as": {
+      "$ref": "common/definitions.json#/trad_as"
+    },
+    "tx_id": {
+      "$ref": "common/definitions.json#/tx_id"
+    },
+    "user_id": {
+      "$ref": "common/definitions.json#/user_id"
+    }
+  },
+  "required": [
+    "exp",
+    "iat",
+    "jti",
+    "tx_id",
+    "account_service_url",
+    "case_id",
+    "collection_exercise_sid",
+    "response_id",
+    "period_id",
+    "ru_ref",
+    "user_id"
+  ],
+  "additionalProperties": false,
```

Comment:
The RAS launch for EQ v3 also includes `return_by` and `survey_id` properties which currently yield the following validation message
```
eq_v3_payload_v1.json - FAILED
[
  {
    instancePath: '',
    schemaPath: '#/additionalProperties',
    keyword: 'additionalProperties',
    params: { additionalProperty: 'survey_id' },
    message: 'must NOT have additional properties'
  },
  {
    instancePath: '',
    schemaPath: '#/additionalProperties',
    keyword: 'additionalProperties',
    params: { additionalProperty: 'return_by' },
    message: 'must NOT have additional properties'
  }
]

Passed: 0 - Failed: 1
```

Reply:
Should they be?. Those two fields aren't in the documentation for launch and runner right now has no knowledge about them so would be dropping them during validation. If they need to be, we need to update the v1/v2 docs.

- `survey_id` exists in the questionnaire schema JSON so do we need RAS to send it?. This `survey_id` from the JSON is then sent downstream.
- Is `return_by` a top level key?. Also, what is the definition?.

Reply:
I guess the only reason RAS would include `survey_id` would be because
- it needs to be included in downstream (and not all published schema have it included)
- its needed by EQ for the schema selection
@martyncolmer may be able to confirm if `survey_id` will always be in published schemas?
If I remember rightly `survey_id` was used for the Census EQ schema selection, but if Business schema selection uses `eq_id` and `form_type` then it may no longer be needed and we can remove it. Let me know and I will raise a card. Ultimately RAS pass it in the claims and therefore the validation will currently fail (as demonstrated).

`return_by` is a business survey key that I believe is rendered to the respondents in various business EQs. It's a top level date defined when collection exercises are authored in ROps. It has the format `YYYY-MM-DD`.

Reply:
The `survey_id` property is a required field in the JSON schema therefore will always exist at the moment.

As for `return_by`, I am happy to add that to the v1/v2 doc. I'll raise a card to add support for that in runner. Do you know what surveys use it so I know when we need it by?.